vocabulary-draft/introduction.md

```diff
@@ -0,0 +1,26 @@
+# Continuous Delivery Events Vocabulary
+
+__Note:__ This is a work-in-progress draft version and is being worked on by members of the Events SIG. You are very welcome to join the work and the discussions!
+
+This document intends to describe a set of events related to different phases of a Continuous Delivery process. 
+These events are categorized by meaning and the phase where they are intenteded to be used. 
+These events are agnostic from any specific tool and are designed to fit a wide range of scenarios. 
+
+The phases covered by this proposal are:
```

Comment:
We might want to have the "orchestration" group (phase?) in here too, but I'd be happy to add that as a follow-up PR.

Reply:
@afrittoli I am not really a fan of having something called "orchestration", first because it means several things for different people and "orchestration" on its own doesn't have any non-technical goal like `Continuous Integration` and `Continuous Delivery`. I think that we should aim for non-technical buckets where the is a clear business value associated with them. I tend to agree that you will end using pipelines for both CI and Continuous Deployment, hence we can have a core or base category where those events go. But I will try to avoid adding technical terms such as orchestration in there. I am not sure what others think about that.

Reply:
@afrittoli I've added the Continuous Delivery bucket to this document and orchestration is part of that